client/src/js/services/util.js

```diff
@@ -96,4 +96,6 @@ function UtilService($filter) {
   service.defaultBirthMonth = '06-01';
   service.minDOB = new Date('1900-01-01');
   service.maxDOB = new Date();
+  //Define the maximum DATE_EMBAUCHE
+  service.maxDateEmbauche = new Date();
```

Comment:
This variable (`maxDateEmbauche`) should be put in the `EmployeeService`.  We will move the `minDOB` and `maxDOB` to the patients service when we get a chance.